app/workers/search/remove_from_index_worker.rb

```diff
@@ -0,0 +1,10 @@
+module Search
+  class RemoveFromIndexWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default, retry: 10
```

Comment:
Can we put this in the `low_priority` queue? I know default has the same priority but I think sticking with the named queues help make it more explicit and clear the importance of the job.